novi-sajt/css/pages/blog.css

```
@@ -0,0 +1,136 @@
+  padding-bottom: 40px;
+  margin-top: -30px;
+  text-align: justify;
+}
+
+.blog-post {
+  display: grid;
+  grid-template-columns: 420px 1fr;
+  gap: 34px;
+  align-items: stretch;
+
+  padding: 34px 0;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)